pkg/kv/iterators_test.go

```diff
@@ -86,17 +97,42 @@ func TestIterators(t *testing.T) {
 		if itr.Err() != nil {
 			t.Fatalf("unexpected error: %v", itr.Err())
 		}
+		itr.Close()
```

Comment:
Can use defer and call it after kv.NewPartitionIterator error check? Relevant for all tests in this file.

Reply:
done